context/app/static/js/components/workspaces/SelectableTemplateGrid/SelectableTemplateGrid.tsx

```diff
@@ -87,11 +107,12 @@ function SelectableTemplateGrid<FormType extends FormWithTemplates>({
         }
       />
       <TemplateGrid
-        templates={templates}
+        templates={sortedTemplates}
         selectItem={selectItem}
         selectedTemplates={selectedTemplates}
         disabledTemplates={disabledTemplates}
         trackingInfo={{ category: WorkspacesEventCategories.WorkspaceDialog }}
+        jobType={jobType.value as string}
```

Comment:
Why is this assertion necessary?

Reply:
Without this, `jobType.value` could potentially be a string array - however I believe the `workspaceJobTypeIdField` defines the field value as a string.

Reply:
Should we file a ticket to resolve this later?